bookkeeper-server/src/main/proto/BookkeeperProtocol.proto

```diff
@@ -73,6 +74,14 @@ message BKPacketHeader {
     required uint64 txnId = 3;
 }
 
+/**
+ * Type of Ledger
+ */
+enum LedgerType {
```

Comment:
I saw `LedgerType` is defined in both `BookkeeperProtocol` and `DataFormats`. Can we define it in a common place? otherwise, it is going to be hard to maintain. 

one thought, if `LedgerType` is associated with metadata, it should exist in `DataFormats`; and `BookkeeperProtocol` references it.

Reply:
Maybe the best option is to use an integer, it will be more simple to handle this in the future.
@sijie  @ivankelly  What do you think ?

Reply:
I am changing this to a top level enum but I am hitting this bug on protoc
https://github.com/google/protobuf/issues/2054

Reply:
I am not sure why do you want to use integer. proto3 handles enum pretty well. proto3 is using forNumber, where valueOf is used?